packages/tizen_bundle/example/lib/bundle_test.dart

```diff
@@ -0,0 +1,255 @@
+import 'package:tizen_log/tizen_log.dart';
+import 'package:test/test.dart';
+import 'package:tizen_bundle/tizen_bundle.dart';
+
+class BundleTest {
```

Comment:
Unit tests should be placed inside the `example/test/` directory. An example app should be an app that visually demonstrates how to use the plugin APIs (e.g. with interactive buttons) but not a simple test runner.

Reply:
Done. It's moved to integration_test.